core/src/mock_alpenglow_consensus.rs

```diff
@@ -543,15 +549,14 @@ impl MockAlpenglowConsensus {
                 if self.prepare_to_receive(s, slot_start).is_err() {
                     error!("Can not initiate mock voting, slot {s} was not released");
                     datapoint_info!("mock_alpenglow", ("runner_stuck", 2, i64), ("slot", s, i64));
-                    return;
```

Comment:
this is the critical change that prevented us from kicking off the runner and thus clearing the state for future use